frappe/utils/caching.py

```diff
@@ -128,3 +128,54 @@ def site_cache_wrapper(*args, **kwargs):
 		return time_cache_wrapper(ttl)
 
 	return time_cache_wrapper
+
+
+def redis_cache(ttl: int | None = None, maxsize: int | None = None) -> Callable:
```

Comment:
lets set ttl by default to 1 hour (or anything else you feel is fine) :grimacing: